src/dc_receive_imf.rs

```diff
@@ -407,6 +407,17 @@ async fn add_parts(
                 .await
                 .unwrap_or_default();
 
+        if chat_id.is_unset() {
+            // check if the message belongs to a mailing list
+            if mime_parser.failed_msg.is_some() {
+                *chat_id = ChatId::new(DC_CHAT_ID_TRASH);
+                info!(
+                    context,
+                    "Message belongs to an NDN list and is not shown in a chat.",
```

Comment:
what is "NDN list" ?

Reply:
if you try to make "NDN" from "Mailing list" but forget to delete "list"
Fixed.